### DISCUSSION TITLE
Is scaled dot attention better than xformers?

### DISCUSSION THREAD

**Discussion by gilroff** [TheLastBen/fast-stable-diffusion#2012] [Q&A] [answered]
I'm just wondering.

**Comment by TheLastBen** [comment, marked as answer]
it fixes the error when generating images greater than 1500x1500

**Reply by buckwheaton** [reply]
Xformers has been updated a couple times since that error started popping up.  Any chance you've been able to test the new versions to see if they result in the same errors?  Before the switch I was able to get noticeably larger image generations without bumping up against the memory limits.
